src/modules/sectionMethods.ts

```diff
@@ -80,7 +87,39 @@ const create = async (data: CreateSection): Promise<Section> => {
         calendarSlot: slot,
     };
 
-    return prisma.section.create({ data: createData });
+    const newSection = await prisma.section.create({ data: createData });
+
+    if (newSection.calendarSlotId) {
+        const exception = await prisma.calendarEventException.findFirstOrThrow({
+            where: { id: newSection.calendarSlotId },
+            include: { eventDetails: true },
+        });
+
+        const interviewer = await prisma.user.findFirstOrThrow({ where: { id: interviewerId } });
+
+        const icalEventDataException = createIcalEventData({
+            id: newSection.calendarSlotId,
+            users: [{ email: interviewer.email, name: interviewer.name || undefined }],
+            start: exception.date,
+            description: restData.description || exception.eventDetails.description,
+            duration: exception.eventDetails.duration,
+            summary: `Interview with ${interview.candidate.name}`,
+            url: `${config.defaultPageURL}/interviews/${interviewId}/sections/${newSection.id}`,
```

Comment:
Don't we have a helper for routes? @tsumo

Reply:
we do, `utils/paths`

Reply:
I'll edit it here and in notifyHr